src/canvas.typ

```diff
@@ -316,13 +316,17 @@
       ctx = r.ctx
       draw-cmds += r.drawables
     }
-
   }
 
   if bounds == none {
     return []
   }
 
+  // Order draw commands by z-index
+  draw-cmds = draw-cmds.sorted(key: (cmd) => {
```

Comment:
This is a really simple way to do this, nice. But is it the best way given all drawables are looped even if the majority of them will be on the 0 layer?

Reply:
I see no problem with that.

Reply:
If you say so :)